### PR TITLE
Sync with master

### PR DESCRIPTION
**Pull request by rvsia**
Merges master to form-state-branch

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
   "private": true,
   "scripts": {
     "bootstrap": "lerna bootstrap",
-    "build": "yarn clean-build && lerna run build --concurrency 1 --ignore @data-driven-forms/react-renderer-demo --ignore @data-driven-forms/pf3-component-mapper",
+    "build": "yarn clean-build && lerna run build --concurrency 1 --ignore @data-driven-forms/react-renderer-demo",
```
Comment (Hyperkid123) [inline package.json:11]: We should ignore the PF3 mapper. The build was not adjusted and we won't continue with development.

File: package.json [hunk 2]
```diff
@@ -28,8 +28,7 @@
   "jest": {
     "verbose": true,
     "testPathIgnorePatterns": [
-      "/node_modules/",
-      "/pf3-component-mapper/"
+      "/node_modules/"
```
Comment (Hyperkid123) [inline package.json:31]: ^^

File: package.json (hunk 1 above)
Comment (rvsia) [reply package.json:11]: there is no pf3 mapper

Comment (Hyperkid123) [reply package.json:11]: Oh the package is gone? I forgot.

**Review by Hyperkid123 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (rvsia) [reply package.json:11]: image

**Pull request merged**